versions.txt

```diff
@@ -0,0 +1,27 @@
+kata_runtime_hash=ea789dbab92d7ae55816c923dbedc4085121fde2
```

Comment:
why this has are needed?

Reply:
This is needed in order to match the source tarball GitHub hash